### ISSUE TITLE
Figure out a way to handle hidden files in SFTP directories

### ISSUE THREAD

**Issue by sseymore** [uoregon-libraries/newspaper-curation-app#64]
Hidden files sometimes get uploaded from Macs. Automatically delete hidden files so NCA doesn't recognize them or mass delete so they don't get uploaded.

**Comment by jechols** [comment]
We could put in a cron job that just auto-deletes these files regularly, or NCA could simply ignore them.  I'm not sure which approach I prefer: cron job would be easier, but ignoring them would mean we don't accidentally delete something that actually matters (say some weird edge case where the file is uploaded as ".foo.pdf" and then renamed to "foo.pdf" only after completion).

**Comment by jechols** [comment]
This is no longer relevant - it was fixed long ago, either on purpose or as a by-product of something else
